src/main/java/org/privacyidea/TokenInfo.java

```diff
@@ -15,192 +15,35 @@
-        this.description = getString(obj, "description");
-        this.failCount = getInt(obj, "failcount");
-        this.id = getInt(obj, ID);
-        this.locked = getBoolean(obj, "locked");
-        this.maxFail = getInt(obj, MAXFAIL);
-        this.otpLen = getInt(obj, OTPLEN);
-        this.resolver = getString(obj, "resolver");
-        this.revoked = getBoolean(obj, "revoked");
-        this.rolloutState = getString(obj, "rollout_state");
-        this.serial = getString(obj, SERIAL);
-        this.syncWindow = getInt(obj, "sync_window");
-        this.tokenType = getString(obj, "tokentype");
-        this.userEditable = getBoolean(obj, "user_editable");
-        this.userID = getString(obj, "user_id");
-        this.userRealm = getString(obj, "user_realm");
-        this.username = getString(obj, USERNAME);
-
-        JsonObject info = obj.getAsJsonObject(INFO);
-        if (info != null) {
-            info.entrySet().forEach(entry -> {
-                if (entry.getKey() != null && entry.getValue() != null) {
-                    this.info.put(entry.getKey(), entry.getValue().getAsString());
-                }
-            });
-        }
-
-        JsonArray arrRealms = obj.getAsJsonArray(REALMS);
-        if (arrRealms != null) {
-            arrRealms.forEach(val -> {
-                if (val != null) {
-                    this.realms.add(val.getAsString());
-                }
-            });
-        }
-    }
-
-    public String getRawJson() {
-        return rawJson;
-    }
-
-    public boolean isActive() {
-        return active;
-    }
-
-    public int getCount() {
-        return count;
-    }
-
-    public int getCountWindow() {
-        return countWindow;
-    }
-
-    public String getDescription() {
-        return description;
-    }
-
-    public int getFailCount() {
-        return failCount;
-    }
-
-    public int getId() {
-        return id;
-    }
-
-    public Map<String, String> getInfo() {
-        return info;
-    }
-
-    public boolean isLocked() {
-        return locked;
-    }
-
-    public int getMaxFail() {
-        return maxFail;
-    }
-
-    public int getOtpLen() {
-        return otpLen;
-    }
-
-    public List<String> getRealms() {
-        return realms;
-    }
-
-    public String getResolver() {
-        return resolver;
-    }
-
-    public boolean isRevoked() {
-        return revoked;
-    }
-
-    public String getRolloutState() {
-        return rolloutState;
-    }
-
-    public String getSerial() {
-        return serial;
-    }
-
-    public int getSyncWindow() {
-        return syncWindow;
-    }
-
-    public String getTokenType() {
-        return tokenType;
-    }
-
-    public boolean isUserEditable() {
-        return userEditable;
-    }
-
-    public String getUserID() {
-        return userID;
-    }
-
-    public String getUserRealm() {
-        return userRealm;
-    }
-
-    public String getUsername() {
-        return username;
-    }
+    boolean active = false;
```

Comment:
These are the attributes of the tokenobject.

So I would call this class/object "Token", since again the "tokeninfo" are arbitrary attributes on the serverside.